pages/common/hydra.md

```diff
@@ -0,0 +1,28 @@
+# hydra
+
+> Password brute forcing tool.
+> More information: <https://github.com/vanhauser-thc/thc-hydra/>.
+
+- Run hydra against an ftp server:
+
+`hydra -l {{Username}} -P {{Password List}} ftp://{{IP Address}}`
+
+- Run hydra against an imap server with plain authentication:
+
+`hydra -L {{Username List}} -p {{Password}} imap://{{IP Address}}/PLAIN`
+
+- Run hydra against a pop3s server. The credentials file should be a list in the form `user:password`:
+
+`hydra -C {{Credentials File}} -6 pop3s://[{{IP v6 Address}}]:{{Port}}/TLS:DIGEST-MD5`
+
+- Run hydra against all ftp servers in an ip range, testing a specific username and password:
+
+`hydra -l {{Useaname}} -p {{Password}} ftp://[{{IP Address}}/{{Subnet}}]/`
+
+- Run hydra against ssh on all a list of targets with every combination usernames and passwords:
+
+`hydra -L {{Username List}} -P {{Password List}} -M {{Target list}} ssh`
+
+- Run hydra against an online login post form:
```

Comment:
I think this is a bit complicated to be in a tldr page. Let us remove this.

Reply:
@agnivade which part of this do you deam too complicated? is it the fact that there are 3 tokens?

I agree that it might be quite complicted but I do use it regularly. This (or a variation on it) is probably the second or third most common hydra command I use

Reply:
@agnivade , I think I might have misunderstood you. Are you refering to the Login Post form or the ssh login with a Target list?

If you were refering to the SSH login, my previous comment is still the case. If you are refering to the Login Post Form then this is the most common hydra command I use. I agree that it is complicated and I have to look it up every time I try and use it.

Reply:
I was referring to the Login Post form.  Note that the tldr project is more of an explainer than a cheatsheet -- in other words, the underlying mission is not to merely provide copy-pastable snippets (like, say, commandlinefu.com) but rather to allow readers to understand the language of the commands, so that the logic behind the options is evident and they can then derive new commands from the provided examples, or compose several simple examples into more complex ones that are useful to them. This is basically the same purpose of man pages, but executed in a more readable way (so we believe), .i.e. through a series of examples.

Reply:
Fair enough. How would feel if i changed it to this: 

```
- Run against an online login form:

`hydra -L {{username_list}} -P {{password_list}} {{ip_address}} http-post-form {{http_service_options}}`

- Show options for a service:

`hydra -U {{service}}`
```

Reply:
Can you give an example of `http_service_options` ? And same for `service`

Reply:
Sure:An example of `http_service_options` is what I had before: `/{{path_to_login_request}}:{{useaname_parameter}}=^USER^&{{password_parameter}}=^PASS^&{{other_parameters}}:{{string_in_failed_responce}}"`

An example of `service` might be `http-post-form` or `ssh`

Reply:
Right, so it leads to the same amount of complexity. It's just hidden. When somebody sees "http_service_options", they will go - well, what does that mean ? And then the whole point of having a tldr page is lost.

We can do the next command. Please use a concrete value like {{ssh}} instead of {{service}}.